docs/introduction.md

```diff
@@ -1,5 +1,5 @@
 
-The OmniPaxos library is mainly driven by the [**OmniPaxos**](omnipaxos/index.md) struct.  It is a plain Rust struct and the user therefore needs to provide a network implementation themselves to actually send and receive messages. In this tutorial we will show how a user should interact with this struct in order to implement a strongly consistent, replicated log. This tutorial will focus on how to use the library and showcase its features. 
+The OmniPaxos library is mainly driven by the [**OmniPaxos**](omnipaxos/index.md) struct. It is a plain Rust struct and the user therefore needs to provide a network implementation themselves to actually send and receive messages. In this tutorial we will show how a user should interact with this struct in order to implement a strongly consistent, replicated log. This tutorial will focus on how to use the library and showcase its features.
```

Comment:
Cross references are a bit special in our case because it gets rendered to the website differently. We need to remove the `.md` suffix. 

More info here: https://github.com/haraldng/omnipaxos/pull/95#discussion_r1200877438